requirements.txt

```diff
@@ -6,10 +6,10 @@
 #
 --no-binary psycopg2
 
-git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
-git+https://github.com/Brown-University-Library/django-shibboleth-remoteuser.git#egg=django-shibboleth-remoteuser
-git+https://github.com/mitodl/mit-moira.git#egg=mit-moira
-git+https://github.com/mitodl/pycaption.git#egg=pycaption
+-e git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
+-e git+https://github.com/Brown-University-Library/django-shibboleth-remoteuser.git#egg=django-shibboleth-remoteuser
+-e git+https://github.com/mitodl/mit-moira.git#egg=mit-moira
+-e git+https://github.com/mitodl/pycaption.git#egg=pycaption
```

Comment:
Hmm. This is a little problematic.

The story about why the `-e` flags were added is a little complicated.
Last week travis builds started to fail, due to failures in `tox`. Tox was failing because when it created a virtualenv it was installing the latest version of pip (pip10). Pip10 would fail unless `git+...` dependency specs were prefixed with `-e`. I do not understand yet why installing a package with the '-e' flag would cause failures on travis.

Unfortunately I couldn't find a way to pin a pip version in tox. So thus this change.

So, we have two options:
1. Update salt deployment to use the `-e` flag.
2. Update the requirements file to satisfy pip 10 in some way that is compatible with salt.

Can you point me towards the salt deployment script?

Reply:
Also, updated default page size to be 50.

Reply:
I'm not sure where the salt deployment script is, but @blarghmatey should be able to help with that.  Ideally it would be best to go back to having the `-e` flags if possible since they're autogenerated with the `requirements.txt` file.